core/storage/statistics/gae_models.py

```diff
@@ -696,6 +893,21 @@ def _shard_answers(
         list, shard the answers such that a list of answer lists are returned.
         The first entry is guaranteed to contain all answers of the current
         answer list.
+
+        Args:
+            current_answer_list: list(Json). list of answer dicts each of which
+            is stored as Json blob.
+            current_answer_list_size: list(int). Number of bytes required
+            to store allthe answers in the current_answer_list.
+            new_answer_list:list(Json). List of new submitted answers each of
+            which is stored as a Json blob.
+
+        Returns:
+            sharded_answer_lists: list(list(Json)): A sharded answer list
```

Comment:
"Returns" can only have one value. In this case it is tuple(typeOfElement1, typeOfElement2) (please fill in the types). @Arunabh98 handled something like this in one of his recent PRs; might be worth asking him for guidance if you're unsure. I'll add him as a reviewer to the PR.

Reply:
@Arunabh98, Can you help me with this?

Reply:
it returns `sharded_answer_lists, sharded_answer_list_sizes`
so, it should be along the lines
```
Returns:
    tuple(list(list(dict)), list(int)). Then explain what each element of tuple is.
```